caiman_napari/mcorr_gui.py

```diff
@@ -30,8 +31,8 @@ def get_params(self, *args, group_params: bool = False) -> dict:
                 'overlaps': (self.ui.spinboxOverlaps.value(), self.ui.spinboxOverlaps.value()),
                 'max_deviation_rigid': self.ui.spinboxMaxDev.value(),
                 'border_nan': 'copy',
-                'decay_time': 0.4,
-                'pw_rigid': True
+                'pw_rigid': self.ui.comboBoxPwRigidBool.currentText(),
```

Comment:
Use a `QCheckBox` instead of a `QComboBox`

Reply:
Would a combo box not be better so the user can select "Rigid" or "Nonrigid" MC as opposed to True/False?